test/parallel/test-crypto-binary-default.js

```diff
@@ -579,6 +579,31 @@ assert.throws(
     name: 'TypeError'
   });
 
+{
+  const hash = crypto.createHash('sha1');
+  hash.on('error', (err) => {
+    assert.strictEqual(err.code, 'ERR_INVALID_ARG_VALUE');
+    assert.strictEqual(err.name, 'TypeError');
+  });
+  hash.end('str', 'hex', (err) => {
+    assert.ok(err != null, `err is ${err} instead of null/undefined`);
```

Comment:
I think we don't want `null/undefined`, the error message is confusing here.
```suggestion
    assert.ok(err != null, `err is null/undefined, expected an Error`);
```
or
```suggestion
    assert.notEqual(err, null);
```